### PR TITLE
Add new setup instructions

### PR DESCRIPTION
**Pull request by junron**
#### Describe the changes you have made in this pr -
Add setup instructions for `new_ux` branch

### CODE REVIEW

**Comment by tachyons** [comment]
We should provide default key for local development and test environment.

File: README.md [hunk 1]
```diff
@@ -38,6 +38,37 @@ Please go through the [Contribution Guidelines](CONTRIBUTING.md) before going fo
 * Run Migrations: `rails db:migrate`
 * At this point, local development can be started with ```rails s -b 127.0.0.1 -p 8080```
 
+### Additional setup instructions for `new_ux`
```
Comment (MrBartusek) [inline README.md]: I think adding 'new_ux' is not needed cause it is on master now

Comment (junron) [reply README.md]: So just 'additional setup instructions'

Comment (MrBartusek) [reply README.md]: @junron I think it might even be a point of setup instructions list

**Pull request merged**